build/prbenchmarks.runtime.windows_arm64.config.yml

```diff
@@ -2,12 +2,9 @@
 components:
     runtime:
         script: |
-            call .\build.cmd clr+libs -c Release -runtimeconfiguration Release -arch arm64
-            call .\src\tests\build.cmd Release arm64 generatelayoutonly /p:LibrariesConfiguration=Release
-            del /F /S /Q .\artifacts\tests\coreclr\windows.arm64.Release\tests\Core_Root\*.pdb
+            call .\build.cmd clr.runtime+clr.alljits+clr.iltools+clr.tools+clr.corelib+clr.nativecorelib -c Release -runtimeconfiguration Release -arch arm64
 
-        arguments:
-            --application.options.outputFiles .\artifacts\tests\coreclr\windows.arm64.Release\tests\Core_Root\
+        arguments: '--{{job}}.options.outputFiles .\artifacts\bin\coreclr\windows.arm64.Release\clrjit.dll --{{job}}.options.outputFiles .\artifacts\bin\coreclr\windows.arm64.Release\coreclr.dll --{{job}}.options.outputFiles .\artifacts\bin\coreclr\windows.arm64.Release\System.Private.CoreLib.dll'
```

Comment:
You'd also need to add

```
variables:
    job: application
```

to the file, if you want to use `{{job}}`

Reply:
The `{{job}}` was added in https://github.com/aspnet/Benchmarks/pull/1754/files in linux-x64 file and I vaguely remember that this is needed as seen on [crank command line reference](https://github.com/dotnet/crank/blob/main/src/Microsoft.Crank.Controller/README.md). This PR just made it consistent to use `{{job}}` everywhere. @sebastienros  - any suggestions?

Reply:
@CarnaViire added it to support the http client benchmark where the application to benchmark is the client (not `application` but `load`). Look at the other files you change and where the is no such comment, if has a 

```
variables:
    job: application
```

You just need to add it before the `profiles:` section

I can do it if you want

Reply:
> I can do it if you want

That will be great.